panels/config/frontend/demo/ha-card-chooser-z.html

```diff
@@ -0,0 +1,35 @@
+<link rel="import" href="./ha-entities-card-z.html">
+
+<script>
+class HaCardChooserZ extends Polymer.Element {
+  static get is() { return 'ha-card-chooser-z'; }
+  static get properties() {
+    return {
+      hass: Object,
+      entity: Object
+    };
+  }
+
+  ready() {
+    super.ready();
+    const hass = this.hass;
+    const entity = this.entity;
+    if (entity.type === 'group') {
+      const states = entity.entities.map(item => hass.states[item.entity_id]);
+      window.hassUtil.dynamicContentUpdater(
```

Comment:
I feel like this component should be named `ha-card-item-renderer` or something. Given an item, it will create the correct web component and pass properties `hass` and `item`. By limiting the logic here, we limit the re-renders.